### PR TITLE
Implement sai port APIs

### PR DESCRIPTION
**Pull request by DanG100**

### CODE REVIEW

File: dataplane/standalone/saiserver/ports.go [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+		}
+		if req.GetAdminState() {
+			stateReq.Operation.AdminStatus = fwdpb.PortState_PORT_STATE_ENABLED_UP
+		}
+		_, err := port.dataplane.PortState(ctx, stateReq)
+		if err != nil {
+			return nil, err
+		}
+	}
+	return nil, nil
```
Comment (alexmasi) [inline dataplane/standalone/saiserver/ports.go:124]: should this return

saipb.SetPortAttributeResponse{}, nil

? I know its basically the same thing

File: dataplane/standalone/saiserver/switch_test.go [hunk 2]
```diff
@@ -102,15 +102,15 @@ func TestCreateSwitch(t *testing.T) {
 	}
 }
 
-func newTestSwitch(t testing.TB) (saipb.SwitchClient, *attrmgr.AttrMgr, func()) {
+func newTestServer(t testing.TB, newSrvFn func(mgr *attrmgr.AttrMgr, srv *grpc.Server)) (grpc.ClientConnInterface, *attrmgr.AttrMgr, func()) {
```
Comment (alexmasi) [inline dataplane/standalone/saiserver/switch_test.go:105]: could newSrvFn be an option in case a test wants to use a default fn like before?

**Review by alexmasi — APPROVED** [review]

File: dataplane/standalone/saiserver/ports.go (hunk 1 above)
Comment (DanG100) [reply dataplane/standalone/saiserver/ports.go]: done

File: dataplane/standalone/saiserver/switch_test.go (hunk 2 above)
Comment (DanG100) [reply dataplane/standalone/saiserver/switch_test.go:105]: made the func optional by passing nil

**Pull request merged**